site/content/en/topics/assumptions.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "Assumptions for Provisioning Tools"
+linkTitle: "Assumptions for Provisioning Tools"
+weight: 1
+type: docs
+description: >
+    Cloud provider assumptions on Azure resources that provisioning tools should take care.
+---
+
+Here is a list of Azure resource assumptions that are required for cloud provider Azure:
+
+* All Azure resources are under the same tenant.
```

Comment:
let's follow RFC 2119 here https://tools.ietf.org/html/rfc2119 

eg. `All Azure resources MUST be under the same tenant.`